src/main/sensors/battery.c

```diff
@@ -116,7 +116,7 @@ PG_RESET_TEMPLATE(batteryConfig_t, batteryConfig,
     .vbatfullcellvoltage = 41,
 
     .vbatLpfPeriod = 35,
-    .ibatLpfPeriod = 40,
+    .ibatLpfPeriod = 10,
```

Comment:
this is not related

Reply:
yea i'll pull that in a separate pull request. thanks for showing me this

Reply:
this is what we do reviews for. like not getting another "telemetry" bug.